docs/src/getting-started/overview.md

```diff
@@ -0,0 +1,180 @@
+    fn send_funds(amount_to_send: u64, recipient_address: b256);
+}
+```
+
+Every Sway file must start with a declaration of what type of program the file contains; here, we've declared that this file is a `library`.
+
+Sway contracts should declare an ABI—an **a**pplication **b**inary **i**nterface in a library so that it can be re-used by downstream contracts. Let's focus on the ABI declaration and inspect it line-by-line.
+
+In the first line, we declare the name of this ABI: `Wallet`. To import this ABI into either a script or another contract for calling the contract, or the contract to implement the ABI, you would use `use wallet_lib::Wallet;`.
+
+In the second line we are declaring an ABI method called `receive_funds` which, when called, should receive funds into this wallet. This method takes no parameters and does not return anything.
+
+_Note that we are simply defining an interface here, so there is no_function body or implementation of the function. We only need to define the interface itself. In this way, ABI declarations are similar to [Rust trait declarations](../advanced/traits.md)._
+
+In the third line we are declaring another ABI method, this time called `send_funds`. It takes two parameters the amount to send, and the address to send the funds to.
+
+### Implementing the ABI Methods in `wallet_contract`
+
+Now that we've defined the interface, let's discuss how to use it. We will start by implementing the above ABI for a specific contract.
+
+To do this, navigate to the `wallet_contract` repo that you created with `forc` previously.
+
+First, you need to import the `Wallet` declaration from the last step. Open up `Forc.toml`. It should look something like this:
+
+```toml
+[project]
+authors = ["user"]
+entry = "main.sw"
+license = "Apache-2.0"
+name = "wallet_contract"
+
+[dependencies]
+core = { git = "http://github.com/FuelLabs/sway-lib-core" }
+std = { git = "http://github.com/FuelLabs/sway-lib-std" }
+```
+
+Include the `wallet_lib` repo by adding the following line to the bottom of the file:
+
+```toml
+wallet_lib = { path = "../wallet_lib" }
+```
+
+Now, open up `main.sw` in `wallet_contract/src` and copy and paste the following code:
+
+```sway
+impl Wallet for Contract {
+    fn receive_funds() {
+    }
+
+    fn send_funds(amount_to_send: u64, recipient_address: b256) {
+    }
+}
+```
+
+This implements the ABI methods with empty bodies. Actual implementation of the bodies is left as an exercise for the reader.
+
+## Build the Contract
+
+Build `wallet_contract` by running
+
+```console
+forc build
+```
+
+from inside the `wallet_contract` directory.
+
+## Deploy the Contract
+
+It's now time to deploy the wallet contract and call it on a Fuel node. You have a few options of how to do the next few steps:
+
+- [Using the Rust SDK](./deploy-and-call-with-rust.md)
+- [Using the Typescript SDK](./deploy-and-call-with-typescript.md)
+- [Using `forc` from the command line](./deploy-and-call-with-cli.md)
```

Comment:
Hmmm. So honestly while users _can_ use the Rust and TS SDKs to `just deploy` their contracts, it seems like way too much complexity. I think we should instead just include instructions here to use the CLI.

My reasoning is that `just deploying` isn't a particularly interesting use of the SDKs, which are more used for either tests or frontends.